src/content/docs/synthetics/synthetic-monitoring/using-monitors/monitor-downtimes-disable-monitoring-during-scheduled-maintenance-times.mdx

```diff
@@ -73,7 +73,7 @@ To delete an existing monitor downtime:
 ## Edit a monitor downtime [#edit]
 
 <Callout variant="tip">
-  If you are unable to edit a window, check your [permissions](/docs/synthetics/new-relic-synthetics/administration/synthetics-permissions-user-groups)
+  If you are unable to edit a window, check your [permissions](/docs/synthetics/synthetic-monitoring/administration/user-roles-synthetic-monitoring/
```

Comment:
You're missing a closing paren there at the end of that URL.

Reply:
Ha, that's what I get for going fast. Fixing now.